### PR TITLE
raftstore, tikv_util: move QueryStats from raftstore to tikv_util component

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What is changed and how it works?

Issue Number: close#13505

What's Changed:

```
Move QueryStats from raftstore to tikv_util component to resolve test_pd_client's dependency on raftstore
```

### Check List 

Tests 

- Unit test
- Integration test

```release-note
None
```

### CODE REVIEW

**Review by haojinming — APPROVED** [review]
LGTM

File: components/tikv_util/src/store/mod.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 TiKV Project Authors. Licensed under Apache-2.0.
```
[the remark this answers is not in the document]
Comment (zeminzhou) [reply components/tikv_util/src/store/mod.rs]: Done, thinks, PTAL~

File: components/tikv_util/src/store/region.rs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    // Because every store can only have one replica for the same region,
+    // so just one round check is enough.
+    lhs.get_peers().iter().all(|lp| {
+        rhs.get_peers()
+            .iter()
+            .any(|rp| rp.get_store_id() == lp.get_store_id() && rp.get_role() == lp.get_role())
+    })
+}
```
[the remark this answers is not in the document]
Comment (zeminzhou) [reply components/tikv_util/src/store/region.rs:39]: Moved, thinks.

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by Connor1996 — APPROVED** [review]

**Pull request merged**